android/initialize.bzl

```diff
@@ -0,0 +1,47 @@
+load("@bazel_common_dagger//:workspace_defs.bzl", "DAGGER_ARTIFACTS", "DAGGER_REPOSITORIES")
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+load("@grab_bazel_common//toolchains:toolchains.bzl", "register_common_toolchains", _buildifier_version = "buildifier_version")
+load("@grab_bazel_common//:workspace_defs.bzl", "GRAB_BAZEL_COMMON_ARTIFACTS")
+load("@grab_bazel_common//tools/buildifier:defs.bzl", "BUILDIFIER_DEFAULT_VERSION")
+# load("@rules_proto//proto:repositories.bzl", "rules_proto_dependencies", "rules_proto_toolchains")
```

Comment:
can we remove this?

Reply:
We need if we need to update Worker protos. Currently Worker protos are precompiled to avoid having a dependency on the compiler for consuming projects.

https://github.com/grab/grab-bazel-common/blob/master/tools/worker/src/main/proto/BUILD.bazel#L1